brave/ui/app/pages/first-time-flow/create-password/import-with-seed-phrase/import-with-seed-phrase.component.js

```diff
@@ -2,8 +2,13 @@ import React from 'react'
 import PasswordWarning from '../password-warning'
 import ImportWithSeedPhrase from '../../../../../../../ui/app/pages/first-time-flow/create-password/import-with-seed-phrase/import-with-seed-phrase.component'
 import { validateMnemonic } from 'bip39'
+import PropTypes from 'prop-types'
 
 module.exports = class BraveImportWithSeedPhrase extends ImportWithSeedPhrase {
+  static contextTypes = {
```

Comment:
Adding `metricsEvent: PropTypes.func,` to this struct will fix the error mentioned in this PR. Since this one takes precedence in the override it won't be available for the non-extended functions